cpp/libcudf_kafka/CMakeLists.txt

```diff
@@ -13,9 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #=============================================================================
-cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
 
-project(CUDA_KAFKA VERSION 0.15.0 LANGUAGES C CXX CUDA)
+project(CUDA_KAFKA VERSION 0.19.0 LANGUAGES C CXX CUDA)
```

Comment:
Can we capture this in the version updater script?

Reply:
We should. I'm not familiar with the version updater script however? Can you point me in the direction of an example?

Reply:
Something in here I think: https://github.com/rapidsai/cudf/blob/branch-0.19/ci/release/update-version.sh#L50